p22/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+pub mod calc;
+
+#[cfg(test)]
+mod tests {
+    use crate::calc::celsius2farenheit;
+    use crate::calc::farenheit2celsius;
+    use crate::calc::fibonacci_loop;
```

Comment:
It's better to keep unit tests near the tested code, i.e. these tests should be in the `calc` module.